src/Bicep.Core.UnitTests/Parsing/ParamsParserTests.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using System.Text;
+using Bicep.Core.Syntax;
+using Bicep.Core.UnitTests.Utils;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bicep.Core.UnitTests.Parsing
+{
+    [TestClass]
+    public class ParamsParserTests
+    {
+        [DataTestMethod]
+        [DataRow("true", "true", typeof(BooleanLiteralSyntax))]
+        [DataRow("false", "false", typeof(BooleanLiteralSyntax))]
+        [DataRow("432", "432", typeof(IntegerLiteralSyntax))]
+        [DataRow("1125899906842624", "1125899906842624", typeof(IntegerLiteralSyntax))]
+        [DataRow("null", "null", typeof(NullLiteralSyntax))]
+        [DataRow("'hello world!'", "'hello world!'", typeof(StringSyntax))]
+        public void LiteralExpressionsShouldParseCorrectly(string text, string expected, Type expectedRootType)
+        {
+            RunExpressionTest(text, expected, expectedRootType);
+        }
+
+        [TestMethod]
+        public void TestDeclaringParams()
+        {
+            var paramIntTest = ParamsParserHelper.ParamsParse("param myint = 12 \n");
+            var paramStringTest = ParamsParserHelper.ParamsParse("param mystr = 'hello world' \n");
+            
+            paramIntTest.Should().BeOfType(typeof(ProgramSyntax));
+            paramStringTest.Should().BeOfType(typeof(ProgramSyntax));
+
+            paramIntTest.Children.OfType<ParameterAssignmentSyntax>().Should().HaveCount(1);
+            paramStringTest.Children.OfType<ParameterAssignmentSyntax>().Should().HaveCount(1);
+        }
+
+        [TestMethod]
+        public void TestUsingKeyword()
+        {
+            var usingTest = ParamsParserHelper.ParamsParse("using './bicep.main' \n");
+
+            usingTest.Should().BeOfType(typeof(ProgramSyntax));
```

Comment:
The assert is not needed. `usingTest` is of type `ProgramSyntax` because that's the return type of `ParamsParserHelper.ParamsParse`.